From: Ted Rodgers <ted.d.rodgers@gmail.com>
Date: Sat, 24 Aug 2024 13:59:07 -0400
Subject: [PATCH 1/2] dual.h: dz -> m_dz -- error: const class
 OSL_v1_13::DualStorage<T has no member named dz; did you mean m_dz

--- a/src/include/OSL/dual.h
+++ b/src/include/OSL/dual.h
@@ -204,7 +204,7 @@ public:
     : m_val(other.m_val)
     , m_dx(other.m_dx)
     , m_dy(other.m_dy)
-    , m_dz(other.dz)
+    , m_dz(other.m_dz)
     {}
 
     OSL_HOSTDEVICE constexpr const T& elem (ConstIndex<0>) const { return m_val; }
